notebooks/notebooks/iclr2022/estimate-flops-adv.ipynb

```diff
@@ -0,0 +1,336 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Computed in the estimate-flops.ipynb notebook
</pre><p><code>estimate-flops-katago.ipynb</code> </p><p> </p><p>Perhaps also mention the cell where it was computed in that notebook: <code>in variable macs_dict</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Updated comment.